### PR TITLE
GF-44, GF-50 - Implement redux logger and environment configuration

### PR DESCRIPTION
**Pull request by iamdavidpetric**

### CODE REVIEW

**Review by MIKALAKE — APPROVED** [review]
Considering your state of health, I won't give you any more trouble, and I'll give you a YES

File: .env.development [hunk 1]
```diff
@@ -0,0 +1,4 @@
+PORT=8080
+REACT_APP_API_PATH_PREFIX='/v1'
+REACT_APP_API_URL=http://localhost:3000
+REACT_APP_ENVIRONMENT=production
```
Comment (iamalextodoran) [inline .env.development]: Empty line at the end of the file for all .env files

File: src/process/redux/projectSlice.js [hunk 2]
```diff
@@ -1,11 +1,12 @@
 import axios from 'axios';
+import baseUrl from '../api/index';
```
Comment (iamalextodoran) [inline src/process/redux/projectSlice.js]: No bueno here.

File: src/show/pages/private/EditProject/index.js [hunk 3]
```diff
@@ -6,7 +6,7 @@ import { useDispatch, useSelector } from 'react-redux';
 import {
   setSelectedProject,
   editProjectAsync,
-  getProjectProjectAsync
+  getProjectAsync
```
Comment (iamalextodoran) [inline src/show/pages/private/EditProject/index.js]: abc

**Review by iamalextodoran — CHANGES_REQUESTED** [review]

File: .env.development (hunk 1 above)
Comment (iamdavidpetric) [reply .env.development]: ok

File: src/process/redux/projectSlice.js (hunk 2 above)
Comment (iamdavidpetric) [reply src/process/redux/projectSlice.js]: kinder bueno?

File: src/show/pages/private/EditProject/index.js (hunk 3 above)
Comment (iamdavidpetric) [reply src/show/pages/private/EditProject/index.js]: cdefg

**Review by iamalextodoran — APPROVED** [review]

**Pull request merged**